tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -11,15 +12,28 @@ namespace DevHome.QuietBackgroundProcesses.UI.ViewModels;
 
 public partial class QuietBackgroundProcessesViewModel : ObservableObject
 {
-    private readonly bool _isFeaturePresent;
     private readonly TimeSpan _zero;
+    private readonly TimeSpan _oneSecond;
+    private ButtonState _buttonState;
 #nullable enable
     private DevHome.QuietBackgroundProcesses.QuietBackgroundProcessesSession? _session;
 #nullable disable
 
+    [ObservableProperty]
+    private bool _isFeaturePresent;
+
     [ObservableProperty]
     private string _sessionStateText;
 
+    [ObservableProperty]
+    private string _quietButtonText;
+
+    private enum ButtonState
+    {
+        Start,
+        Stop,
```

Comment:
I suggest renaming these to "Started" and "Stopped" to be extra clear about the state, rather than using verbs.